tests/phpunit/tests/block-supports/dimensions.php

```diff
@@ -0,0 +1,104 @@
+<?php
```

Comment:
Whoops forgot to share the naming convention:

For the file:
It's the function's name in camelCase without the `_`.

For the test:
It's in this format:
```
Tests_{group}_{functionNameInCamelCase}
```

See handbook here https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#test-classes for more information.

Note: In Gutenberg, more work is needed in the linting before its tests can be converted into Core's coding standards.

Reply:
@hellofromtonya According to the handbook, it says the test class should be in this format:
```
Tests_{Group}_{FunctionNameInTitleCase}
```

> Thus, the class `Tests_Comment_GetCommentClass()`, which contains tests for the `get_comment_class()` function, is located in `tests/comment/getCommentClass.php`.

Reply:
@costdev The other test in the group has it as `Block_Supports`. So I felt it that way for consistency and because the handbook is unclear if the group should also be smooched together. Thinking that topic needs a Trac ticket of its own to discuss and get consensus.

Reply:
Regarding `_Wp` rather than `_wp`, see [this ticket](https://core.trac.wordpress.org/ticket/56846) with support for encouraging `Wp` for consistency among all test classes, despite there being more cases of `_wp` in the test suite historically.

Reply:
Oh wait I do see what you mean though about capitalizing the first letter of the function's name. Yup that needs changing. Good catch. Will do that in the commit.

Reply:
Capitalization done here https://github.com/WordPress/wordpress-develop/pull/3940/commits/12cebb4cd4239b4d9c06a05c20ccd2c21b5f55fb. Thanks @costdev!